migrator/init/user_for_api.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+psql -c "CREATE USER gateway WITH PASSWORD $UM_PASSWORD;"
```

Comment:
WDYT about renaming user 'gateway' to 'um_user'?
Pattern: [app abbreviation] + '_user

Reply:
Agree, fix it